Community Toolbox/scripts/utils_Math/utils_Math.gml

```diff
@@ -24,3 +26,61 @@ function eucdiv(_dividend, _divisor) {
     _dividend -= eucmod(_dividend, _divisor);
     return _dividend div _divisor;
 }
+
+#endregion
+
+#region Reaching target values
+
+/// @func lerp_angle(from,to,amount)
+/// @desc Calculates the angle interpolated between a starting and destination angle for the given fraction amount.
+/// @arg {Real} from            The starting angle.
+/// @arg {Real} to              The destination angle.
+/// @arg {Real} amount          The amount to interpolate (0 matches the starting angle, 1 matches the destination angle).
+/// @returns {Real}
+function lerp_angle(_from, _to, _amount) {
+    var _new_angle = _from + angle_difference(_to, _from) * _amount;
+    return eucmod(_new_angle, 360);
+}
+
+/// @func approach(current,target,[step])
+/// @desc Calculates the value a step closer to the target value. If the target is close enough, returns the target.
+/// @arg {Real} current         The current value.
+/// @arg {Real} target          The target value.
+/// @arg {Real} [step]          The step to approach the target by (1 by default).
+/// @returns {Real}
+function approach(_current, _target, _step = 1) {
+    // for a negative step, the value moves away from the target indefinitely
+    if (_step < 0)
+        return _current + (_target <= _current ? -_step : _step);
```

Comment:
Nit: This can be done with `sign`, for consistency with line 60.

Reply:
Changed to use `sign`